README.md

```diff
@@ -163,417 +152,42 @@ Oban requires Elixir 1.14+, Erlang 23+, and PostgreSQL 12.0+ or SQLite3 3.37.0+.
-
-* Each queue will run as many jobs as possible concurrently, up to the
-  configured limit. Make sure your system has enough resources (i.e. database
-  connections) to handle the concurrent load.
-
-* Queue limits are local (per-node), not global (per-cluster). For example,
-  running a queue with a local limit of one on three separate nodes is
-  effectively a global limit of three. If you require a global limit you must
-  restrict the number of nodes running a particular queue.
-
-* Only jobs in the configured queues will execute. Jobs in any other queue will
-  stay in the database untouched.
-
-* Be careful how many concurrent jobs make expensive system calls (i.e. FFMpeg,
-  ImageMagick). The BEAM ensures that the system stays responsive under load,
-  but those guarantees don't apply when using ports or shelling out commands.
-
-## Defining Workers
-
-Worker modules do the work of processing a job. At a minimum they must define a
-`perform/1` function, which is called with an `%Oban.Job{}` struct.
-
-Note that the `args` field of the job struct will _always_ have string keys, regardless of the key
-type when the job was enqueued. The `args` are stored as JSON and the serialization process
-automatically stringifies all keys. Also, because `args` are always encoded as JSON, you must
-ensure that all values are serializable, otherwise you'll have encoding errors when inserting jobs.
-
-Define a worker to process jobs in the `events` queue:
-
-```elixir
-defmodule MyApp.Business do
-  use Oban.Worker, queue: :events
-
-  @impl Oban.Worker
-  def perform(%Oban.Job{args: %{"id" => id} = args}) do
-    model = MyApp.Repo.get(MyApp.Business.Man, id)
-
-    case args do
-      %{"in_the" => "business"} ->
-        IO.inspect(model)
-
-      %{"vote_for" => vote} ->
-        IO.inspect([vote, model])
-
-      _ ->
-        IO.inspect(model)
-    end
-
-    :ok
-  end
-end
-```
-
-The `use` macro also accepts options to customize `max_attempts`, `priority`, `tags`, `unique`,
-and `replace` options:
-
-```elixir
-defmodule MyApp.LazyBusiness do
-  use Oban.Worker,
-    queue: :events,
-    priority: 3,
-    max_attempts: 3,
-    tags: ["business"],
-    unique: true,
-    replace: [scheduled: [:scheduled_at]]
-
-  @impl Oban.Worker
-  def perform(_job) do
-    # do business slowly
-
-    :ok
-  end
-end
-```
-
-Like all `use` macros, options are defined at compile time. Avoid using `Application.get_env/2` to
-define worker options. Instead, pass dynamic options at runtime by passing them to
-`MyWorker.new/2`:
-
-```elixir
-MyApp.MyWorker.new(args, queue: dynamic_queue)
-```
-
-Successful jobs should return `:ok` or an `{:ok, value}` tuple. The value returned from
-`perform/1` is used to control whether the job is treated as a success, a failure, cancelled or
-deferred for retrying later.
-
-See the `Oban.Worker` docs for more details on failure conditions and `Oban.Telemetry` for details
-on job reporting.
+## Quick Getting Started
```

Comment:
Now, we have this section which kind of "shows off" Oban, but then you have real docs and guides for getting into more detail. The README got real slim with this.